website/src/index.html

```diff
@@ -395,26 +335,15 @@ <h1>PRIZES</h1>
           </div>
         </div>
 
-        <div class="row" id="register">
-          <div class="col">
-            <h1>REGISTER</h1>
-            <a
-              href="https://docs.google.com/forms/d/17rF3REYs57XSFeE7xyCcl1HgG0Hwrc0vHmLSwUoJ6Nc/viewform?ts=626861a8&edit_requested=true">Register
-              here</a>.
-          </div>
-        </div>
-
         <div class="row" id="submit">
           <div class="col">
             <h1>SUBMIT A SOLUTION</h1>
+
+
             <ul>
-              <li>Track 1: <a
-                  href="https://docs.google.com/forms/d/1QtIOvEBt19aHi0Yreh4c5dPKObpwEkoy-NfAB7D2qEo/viewform?ts=624b50d2&edit_requested=true">Google
-                  Form</a></li>
-              <li>Track 2: <a
-                  href="https://docs.google.com/document/d/1ojtd5r42Jl_zNl79BdEilQaM3wt9OWoZLQWXQhInq-Q/edit?usp=sharing">
-                  Submission and Review Template
-                </a></li>
+              <li>Track 1: Read the submission FAQs. Submit your code here:<a href="https://docs.google.com/forms/d/1QtIOvEBt19aHi0Yreh4c5dPKObpwEkoy-NfAB7D2qEo/viewform?ts=624b50d2&edit_requested=true">Google Form</a></li>
+              <li>Track 2: Read the <a href="https://docs.google.com/document/d/1ojtd5r42Jl_zNl79BdEilQaM3wt9OWoZLQWXQhInq-Q/edit?usp=sharing" target="blank">submission essay guidelines and jury review template</a>. Submit your code and essay here: <a href="https://docs.google.com/forms/d/1QtIOvEBt19aHi0Yreh4c5dPKObpwEkoy-NfAB7D2qEo/viewform?ts=624b50d2&edit_requested=true">Google Form</a></li>
```

Comment:
Track 2: Submit code in [Google Form 1] and submit your essay in [Google Form 2]. 

We should submit all code to the same Google Form because they'll use the same evaluation loop.